main.tf

```diff
@@ -0,0 +1,127 @@
+locals {
+  app_env          = data.terraform_remote_state.common.outputs.app_env
+  app_environment  = data.terraform_remote_state.common.outputs.app_environment
+  app_name_and_env = "${var.app_name}-${local.app_env}"
+  ecs_cluster_arn  = data.terraform_remote_state.common.outputs.ecs_cluster_id
+  name_tag_suffix  = "${var.app_name}-${var.customer}-${local.app_environment}"
+}
+
+/*
+ * Create cloudwatch log group for app logs
+ */
+resource "aws_cloudwatch_log_group" "ytbackup" {
+  name              = local.app_name_and_env
+  retention_in_days = 60
+
+  tags = {
+    name = "cloudwatch_log_group-${local.name_tag_suffix}"
+  }
+}
+
+/*
+ * Create task definition for file system backup
+ */
+locals {
+  task_def = templatefile("${path.module}/task-def-yt-backup.tftpl",
```

Comment:
For the curious, [here's a way](https://github.com/silinternational/ethnologue-cam/blob/8c14f978212925a03ba71a1bfc333f6f4d55ce39/terraform/main.tf#L138) to create a task definition without using a template. In some ways it is simpler, but make your own opinion.